autoload/SpaceVim/plugins.vim

```diff
@@ -191,6 +191,7 @@ let s:plugins.lang = [
             \ ['isundil/vim-irssi-syntax',               { 'on_ft' : 'irssi'}],
             \ ['lervag/vimtex',                          { 'on_ft' : 'tex'}],
             \ ['vimperator/vimperator.vim',              { 'on_ft' : 'vimperator'}],
+            \ ['fatih/vim-go',                           { 'on_ft' : 'go'}],
```

Comment:
need loadconf  or loadconf_before if the config file need to be loaded before vim-go.